src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -114,7 +114,7 @@ class NewConsentGroup extends Component {
         .then(resp => {
           // TODO: window.location.href is a temporal way to redirect the user to project's consent-group page tab. We need to change this after
           // transitioning from old gsps style is solved.
-          window.location.href = [this.props.serverURL, projectType, "show", this.props.projectKey, "?tab=consent-groups&new"].join("/");
+          window.location.href = [this.props.serverURL, "project", "main?projectKey=" + this.props.projectKey + "&tab=consent-groups&new"].join("/");
```

Comment:
I'm a little sad that we still need `window.location.href`, but I understand it. Hopefully we can get away from this when more of the app is ported over to react.